snp_automate_2022/urdf/workcell.xacro

```diff
@@ -142,7 +142,7 @@
     <child link="ee"/>
   </joint>
 
-  <xacro:property name="camera_cal" value="${load_yaml('$(find snp_automate_2022)/config/camera_calibration.yaml')}" />
+  <xacro:property name="camera_cal" value="${xacro.load_yaml('$(find snp_automate_2022)/config/camera_calibration.yaml')}" />
```

Comment:
Does this work in foxy?

Reply:
Apparently not.

Reply:
After offline discussion, added an argument in the xacro where the launch file passes in the `ROS_DISTRO` env variable and a check is performed to see how the `load_yaml` method should be called.